tools/maplint/lints/unary_vents.yml

```diff
@@ -0,0 +1,5 @@
+/obj/machinery/atmospherics/unary:
+  banned_neighbors:
+  - /obj/machinery/atmospherics/pipe/manifold/hidden/supply
+  - /obj/machinery/atmospherics/pipe/simple/hidden/scrubbers
```

Comment:
this should probably be all pipes

Reply:
Just porting the tests for now, can tighten them up later